README.md

```diff
@@ -116,6 +118,111 @@ while let Some(batch) = stream.next().await {
 }
 ```
 
+### TLS support
+You can add TLS support to your consumer or producer for secured communication. To enable this, start with specifying the `TlsConnectionOptions`,
+and pass it into an instance of the `ProducerBuilder` or `ConsumerBuilder`.
+
+Example for Producer with TLS support:
+```rust
+let tls_option = TlsConnectionOptions {
```

Comment:
On this one and all subsequent tlsConnection options, use something like:
```
let tls_option = samsa::prelude::TlsConnectionOptions {
        broker_options: vec![samsa::prelude::BrokerAddress {
            host: "127.0.0.1:9092".to_owned(),
            port: 9092,
        }],
        key: "/path_to_key_file".into(),
        cert: "/path_to_cert_file".into(),
        cafile: Some(("/path_to_ca_file").into()),
    };
```

Reply:
Okay, this makes sense. I hesitated to use the `.into()` method because I wanted to show that it was a BufPath. I will make the changed now

Another question: does this BrokerAddress class apply to the non-TLS variants?

Reply:
Yes, to make it consistent across both. Each connection type requires that BrokerAddress at a minimum